src/NLog/Logger.cs

```diff
@@ -82,10 +82,14 @@ protected internal Logger()
         /// </summary>
         public LogFactory Factory { get; private set; }
 
+#if !NET3_5 && !NET4_0 && !SILVERLIGHT && !WINDOWS_PHONE
```

Comment:
Maybe just change to `NET4_5` ?

Reply:
I prefer a blacklist, as that will work better for the future